src/hooks/useStockInfo.jsx

```diff
@@ -0,0 +1,39 @@
+import React, {createContext, useContext, useState} from 'react'
+import {stockInfo} from "../shared/stockInfo";
+
+const StockInfoContext = createContext("");
+
+const StockInfoProvider = ({children}) => {
```

Comment:
Please pull the latest changes 
I am not sure why this file is here when it is already present under the contexts folder